### PR TITLE
Made RobotMap

### PR DESCRIPTION
**Pull request by stardust4089**
Created a robot-map to be able to make it easy to access and modify the values and addresses for the intake subsystem. Made by @GreenMachineStudios

### CODE REVIEW

File: src/main/java/frc/robot/RobotMap.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+
+public class RobotMap {
+    public static final class Pneumatics {
+        public static final int PCM = 41;
+        public static final PneumaticsModuleType PCM_TYPE = PneumaticsModuleType.CTREPCM;
```
Comment (silverlightning926) [inline src/main/java/frc/robot/RobotMap.java:8]: Are you sure that this is the PCM type we are planning on using this year?

File: src/main/java/frc/robot/RobotMap.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        public static final int INTAKE_DEPLOY_PCM_CHANNEL = 0;
```
Comment (silverlightning926) [inline src/main/java/frc/robot/RobotMap.java:9]: Remember to change this one you have this setup on the actual bot

File: src/main/java/frc/robot/RobotMap.java (hunk 1 above)
Comment (stardust4089) [reply src/main/java/frc/robot/RobotMap.java:8]: Yes I'm sure that's it the correct PCM

File: src/main/java/frc/robot/RobotMap.java (hunk 2 above)
Comment (stardust4089) [reply src/main/java/frc/robot/RobotMap.java:9]: Yes

**Review by silverlightning926 — APPROVED** [review]
Looks good to me!

**Review by SamarthRao91 — APPROVED** [review]
Looks good!

**Pull request merged**